CHANGES/5431.misc

```diff
@@ -0,0 +1 @@
+Syncing custom/unknown repository metadata.
```

Comment:
I would also make it a feature and not just misc.
Users don't care much about models and they might not fully understand what publish is, however they know very well the term "sync" and  it's a feature that  they can perform now sync (and publish is kinda implied :) for them)

Reply:
Sure, works for me